build/tools/amebasmart/gnu_utility/rtk_data_binary.py

```diff
@@ -1,4 +1,21 @@
 #!/usr/bin/env python
+###########################################################################
```

Comment:
minor comment, this change doesn't seem to fit with the commit. Could it be moved to a new commit?

Reply:
done~